src/components/options.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+
+class OptionsComponent extends React.Component {
+
+    constructor(props) {
+        super(props)
+    }
+
+    renderSelect(choice, e) {
+        return (
+            <select className="uk-select" onChange={(e) => this.props.onChoice(e.target.value)}>
+            {this.renderChoices()}
+            </select>
+        )
+    }
+
+    renderChoice(choice) {
+        return (
+            <option value={choice}>
+                {choice}
+            </option>
+        )
+    }
+
+    renderChoices() {
+        return this.props.choices.map(choice => this.renderChoice(choice))
+    }
+
+    render(choice) {
+        return (
+            <form>
+                {this.renderSelect(choice)}
+            </form>
+        )
+    }
+
+}
+
+
+export default OptionsComponent
```

Comment:
I love it!

The comments below are not for now, but they are for you to look at after merging.

One thing you may want to consider is to allow the choices property to be an array or an object, that keys and messages can be mapped to. For example:

```js
let choices = {
  "FAULTY": "The product has a fault or defect",
  "SIZE_TOO_SMALL": "The product is too small for me to wear",
  "WRONG_ITEM": "The wrong product was in the box"
}

// ....

<OptionsComponent
    choices={choices}
    onChoice{(choice) => console.log("The selected return reason code was", choice)} />

// If you pick the wrong item dropdown, you'll get:
// > "The selected return reason code was WRONG_ITEM"
```

To make this work, you would need to update renderChoices() to [map an object](https://stackoverflow.com/a/14810722) rather than an array, and when you call `this.renderChoice()` on line 27, you would provide the value and the message. You would then update the `renderChoice()` function to put the value in the `<option value={value}>`, separately from the message.